.travis/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+
+name=$1
+health=$2
+options=$3
+
+docker run --detach --name=${name} --health-cmd="${health}" ${options} > /dev/null
```

Comment:
in an isolated environment like Travis CI `--rm` is not necessary but should help when testing the script locally